python/src/FunctionImplementation_doc.i.in

```diff
@@ -623,9 +623,9 @@ OT_Function_getParameterDimension_doc
 "Draw the output of function as a :class:`~openturns.Graph`.
 
 Available usages:
-    draw(*inputMarg, outputMarg, CP, xiMin, xiMax, ptNb*)
+    draw(*inputMarg, outputMarg, a, xiMin, xiMax, ptNb*)
```

Comment:
This change is inconsistent with what follows (CP is renamed centralPoint).

Reply:
This was already fixed at 8b6c8bca1ecc762b7f247cf40bcfc5b37272dbf4